examples/tests/valid/subdir-3/blah.txt

```diff
@@ -0,0 +1,2 @@
+even though this directory name is invalid for bringing we will
```

Comment:
we should check that something like:

```
good_dir/my.w
good_dir/bad-dir/nothing.txt
```

then:

```js
bring "./good_dir" as g;
```

should work (because the `bad-dir` is ignored).

Reply:
Yupp, this scenario is now tested